docs/framework/contributing/development-environment.md

```diff
@@ -226,17 +226,9 @@ In addition to the possibility of defining exclusions in the `<meta>` tag, it is
 <a href="path/to/page" data-cke-crawler-skip>No entry for crawler, sorry</a>
 ```
 
-## Generating content styles
+## Generating editor styles
 
-It is possible to generate a style sheet containing content styles brought by all CKEditor&nbsp;5 features. To do that, execute:
-
-```
-yarn docs:content-styles
-```
-
-The style sheet will be saved in the `build/content-styles` folder.
-
-To learn more, refer to the {@link getting-started/advanced/content-styles Content styles} guide.
+It is possible to generate a style sheet containing content styles brought by all CKEditor&nbsp;5 features. To obtain such a style sheet, follow our {@link getting-started/advanced/integrating-from-source-webpack#option-extracting-css CSS extracting} guide. To learn more, refer to the {@link getting-started/advanced/content-styles Content styles} guide.
```

Comment:
This is a legacy guide. Should we link to legacy guides? If the mechanism is valid, maybe we should extract it to current documentation to keep (the legacy guides will be purged in the foreseeable future).

Reply:
I've also linked to this section in `Sharing content styles between frontend and backend` as otherwise I don't see how could this section make any sense.

Reply:
The end result would be pretty much the warning that this is legacy, and then the section we copied from the new docs page. We might as well just redirect...

Reply:
If it's valid, why don't we move it somewhere or split as a new guide? If it's invalid for NIM anymore, let's not mention it at all.

Reply:
By the way, here's a moder CSS guide in NIM docs: https://ckeditor.com/docs/ckeditor5/latest/getting-started/setup/css.html